src/testing/cluster.zig

```diff
@@ -499,7 +501,9 @@ pub fn ClusterType(comptime StateMachineType: anytype) type {
                             replica.tick();
 
                             // For performance, don't run every tick.
-                            if (i % 100 == 0) JournalChecker.check(replica);
+                            if (cluster.prng.chance(ratio(1, 100))) {
+                                JournalChecker.check(replica);
+                            }
```

Comment:
This one feels like it'll also make VOPR massively faster.

Reply:
:facepalm: Ahh excellent catch on this typo!